pkgs/applications/terminal-emulators/iterm2/default.nix

```diff
@@ -42,7 +48,7 @@ stdenv.mkDerivation rec {
     homepage = "https://www.iterm2.com/";
     license = licenses.gpl2;
     maintainers = with maintainers; [ tricktron ];
-    platforms = platforms.darwin;
+    platforms = [ "x86_64-darwin" ];
```

Comment:
Does this change prevent `iterm2` running on Apple silicon?

Reply:
Oh if I keep platforms.darwin, the GitHub Action for this PR will fail.
Seems it's dependence missing on aarch64-darwin, so I delete it.

Reply:
yes

Reply:
I reverted this line to:

```
platforms.darwin
```

and it built fine on my M1 machine. If we could leave this to include apple silicon that would be great!